### ISSUE TITLE
Headline "Test interfaces, not classes" is misleading

### ISSUE THREAD

**Issue by gerhard-dierkes-sap** [SAP/styleguides#134]
Proposal: "Test code under test via interface"

**Comment by mAltr** [comment]
Common is the statement "code against interfaces, not implementations" (Practical API Design, p87) , so may we could use this and rephrase it to "Test againt interfaces, not implementations". 
Would this be make more sense?

**Comment by gerhard-dierkes-sap** [comment]
Yes, having the referenced statement in mind, I think it is fine to have "Test against interfaces, not implementations".

**Comment by HrFlorianHoffmann** [comment]
Merged the change, so maybe close this issue?

**Comment by gerhard-dierkes-sap** [comment]
Thanks!
